### DISCUSSION TITLE
Why don't I have any color?

### DISCUSSION THREAD

**Discussion by LintaoAmons** [folke/flash.nvim#20] [Q&A] [answered]
![image](https://github.com/folke/flash.nvim/assets/95092244/227463eb-fc9b-405a-b42b-0caff9416c2b)

**Comment by folke** [comment, marked as answer]
Your theme doesn't differentiat between `IncSearch` and `Substitue`.

You can find the full list of hl groups here https://github.com/folke/flash.nvim#-highlights

You can also configure different groups to be used. See the config options
